sources/platform/integrations/actors/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Actors
+description: Learn how to integrate with other Actors and tasks
+sidebar_position: 11.10
+slug: /integrations/actors
+---
+
+> This is an experimental feature and currently available on request. Ask our support to enable it for your account.
```

Comment:
maybe link to support

Reply:
Also, this is experimental for developers or end users? If I understand correctly, it's experimental for developers. But once the integration is published, any end user can use it. Is that correct?

Reply:
Partially, depending on how is the integration published. If as a specific or generic integration, then it's true, but the option to connect any actor is hidden behind the feature flag.